tests/libnet/ipaddress.go

```diff
@@ -0,0 +1,29 @@
+package libnet
+
+import (
+	v1 "k8s.io/api/core/v1"
+	netutils "k8s.io/utils/net"
+)
+
+func GetPodIpByFamily(pod *v1.Pod, family v1.IPFamily) string {
+	var ips []string
+	for _, ip := range pod.Status.PodIPs {
+		ips = append(ips, ip.IP)
+	}
+	return getIp(ips, family)
```

Comment:
i meant imo there is no 2 functions here, and that you can just range pod.Status.PodIPs
and then use ip.IP

Reply:
+1; is there any advantage of converting all the IPs to strings before getting the IP by family ? ( I understand we're not talking about N IPs, only 2, but still ..)

Reply:
What was in my mind is that when we will have to add `GetVmIpByFamily` we will reuse - `getIp`.
We will need `GetVmIpByFamily` when moving the dual stack test to the `DescribeTable`\test per ip family approach.

Reply:
Right. Makes sense. Didn't know / remember that the `vmi.Status.Interfaces[i].IP` was a string.

Reply:
as was mentioned in the PR before, so far we dont have it, do we still want "air conditioner preparation" ? (how to tell this idiom in english please?)